app/src/main/res/layout/fragment_exposure_notification_list.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <data>
+        <import type="android.view.View" />
+        <variable name="model" type="fi.thl.koronahaavi.exposure.ExposureDetailViewModel" />
+    </data>
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <androidx.core.widget.NestedScrollView
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:padding="20dp"
+            android:clipToPadding="false">
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="vertical">
+
+                <TextView
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="10dp"
+                    android:text="@string/notification_list_title"
+                    android:textAppearance="?attr/textAppearanceHeadline2"
+                    android:hyphenationFrequency="normal" />
+
+                <TextView
+                    android:id="@+id/text_notification_list_last_check"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="10dp"
+                    android:textAppearance="?attr/textAppearanceBody2"
+                    tools:text="@tools:sample/lorem"/>
+
+                <androidx.recyclerview.widget.RecyclerView
```

Comment:
One can see the overscroller effect in the recyclerview area and in the bottom of the bottom sheet (depending on where the touch point is) - seeing it in the "middle" of a view is slightly buggy looking. Would `android:overScrollMode="never"` help?

I'm also a bit curious about using RecyclerView in this view in the first place - is it the easiest / most convenient way of implementing this view / would it be too complicated to have the recycler view higher in the hierachy + having the textviews as its items / could one just add the items to the LinearLayout?

Reply:
Thanks, overscroll mode="never" removes the effect
And yes, recyclerview is just convenient I guess, and it's nice that its adapter provides dynamic additions and removals of list items, even thought it might not be necessary to have this update dynamically